.github/workflows/wheels.yml

```diff
@@ -22,9 +22,10 @@ on:
     #        │  │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
     #        │  │ │ │ │
     - cron: "42 2 * * SUN,WED"
-  push:
```

Comment:
are there any circumstances where a push commit would be useful, over the manual start?

Reply:
I think only maintainers can push the _Run workflow_ button, so that may be useful to keep for now. `[wheel build]` isn't super pretty, but at least it doesn't cause any problems.